### DISCUSSION TITLE
TimeDomainSpecAugment leading to nan

### DISCUSSION THREAD

**Discussion by matthewkperez** [speechbrain/speechbrain#1693] [Q&A]
Hello,
I've adapted the librispeech recipe to run using my private dataset using the wav2vec_hf_params.yaml
However, when training I encounter a `Loss is nan` error that occurs after applying augmentation to the training data.

Attaching the relevant files below
Cheers,

[wav2vec_hf_params_yaml.txt](https://github.com/speechbrain/speechbrain/files/9965373/wav2vec_hf_params_yaml.txt)
[train_with_wav2vec_py.txt](https://github.com/speechbrain/speechbrain/files/9965378/train_with_wav2vec_py.txt)
[log.txt](https://github.com/speechbrain/speechbrain/files/9965399/log.txt)

**Comment by matthewkperez** [comment]
Update: I fixed the error by running with DDP rather than DP following [https://github.com/speechbrain/speechbrain/issues/1580](url)

**Reply by TParcollet** [reply]
@anautsch please consider adding the "drop DP support" to our next general meeting 😃

**Reply by Adel-Moumen** [reply]
https://github.com/speechbrain/speechbrain/issues/1713 another one :-) @TParcollet @anautsch
